virtualizarr/tests/test_manifests/test_store.py

```diff
@@ -83,8 +122,28 @@ def _generate_manifest_store(
         fill_value=0,
     )
     manifest_array = ManifestArray(metadata=array_metadata, chunkmanifest=manifest)
+    scalar_chunk_manifest = ChunkManifest.from_arrays(
+        paths=np.array(f"{prefix}/{filepath}", dtype=np.dtypes.StringDType),
```

Comment:
Similar to my other comment, I suggest you just add a `# type: ignore` comment on this line.